### PR TITLE
version bump: dev-libs/libuv-1.6.0

### PR DESCRIPTION
**Pull request by jbergstroem**
Copy from libuv 1.5.0.

### CODE REVIEW

**Comment by jbergstroem** [comment]
we'll probably see a 1.6.1 shortly -- let's hold this for now.

**Comment by mgorny** [comment]
Ok, held it :).

**Comment by jbergstroem** [comment]
Didn't mean we should close it, rather just wait until 1.6.1 was out (which was roughly 5 days ago). Reopening and renaming.

Edit: can you reopen? Just pushed another commit; guessing it doesn't update until the PR is open.

**Comment by mgorny** [comment]
Let's see.

File: dev-libs/libuv/libuv-1.6.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+LICENSE="BSD BSD-2 ISC MIT"
+SLOT="0/1"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux"
+IUSE="static-libs"
+
+DEPEND="sys-devel/libtool
+	virtual/pkgconfig[${MULTILIB_USEDEP}]"
+
+src_prepare() {
+	echo "m4_define([UV_EXTRA_AUTOMAKE_FLAGS], [serial-tests])" \
```
Comment (mgorny) [inline dev-libs/libuv/libuv-1.6.1.ebuild:22]: What's the deal with this? Broken release? Upstream bug? :P

File: dev-libs/libuv/libuv-1.6.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+multilib_src_configure() {
+	local myeconfargs=(
+		cc_cv_cflags__g=no
+		$(use_enable static-libs static)
+	)
+	autotools-utils_src_configure
+}
+
+multilib_src_test() {
+	mkdir "${BUILD_DIR}"/test || die
```
Comment (mgorny) [inline dev-libs/libuv/libuv-1.6.1.ebuild]: multilib_\* phases start in BUILD_DIR.

File: dev-libs/libuv/libuv-1.6.1.ebuild (hunk 1 above)
Comment (jbergstroem) [inline dev-libs/libuv/libuv-1.6.1.ebuild:22]: Was here before I started working on the ebuild. I'll revisit running tests without and see how it goes.

Edit: the build starts whining. I'll leave it in for now.

Comment (mgorny) [inline dev-libs/libuv/libuv-1.6.1.ebuild:22]: Oh, now I see what it does. Ok then.

**Comment by mgorny** [comment]
```
+*libuv-1.6.1 (15 Jun 2015)
+
+  15 Jun 2015; Michał Górny <mgorny@gentoo.org> +libuv-1.6.1.ebuild:
+  Version bump. https://github.com/gentoo/gentoo-portage-rsync-mirror/pull/141
+  by jbergstroem.
```

**Comment by hasufell** [comment]
Can you please ping me on such pull requests? I am not interested in other random PRs on this repository, so I am not watching

**Comment by jbergstroem** [comment]
@hasufell ok, will do

**Pull request closed without merging**